.github/CODEOWNERS

```diff
@@ -0,0 +1,181 @@
+.github/workflows/** @llvm/pr-subscribers-pr-subscribers-github-workflow
+/clang/*/Driver/ @llvm/pr-subscribers-clang-driver
+/compiler-rt/*/*san* @llvm/pr-subscribers-sanitizer
+/compiler-rt/*/xray/ @llvm/pr-subscribers-xray
+/llvm/*/LTO/ @llvm/pr-subscribers-llvm-lto
+/llvm/*/Linker/ @llvm/pr-subscribers-llvm-lto
+/llvm/*/MC/ @llvm/pr-subscribers-llvm-mc
+/llvm/*/ThinLTO/ @llvm/pr-subscribers-llvm-lto
```

Comment:
Sorry, request another line
`/llvm/tools/gold/ @llvm/pr-subscribers-llvm-lto`

Reply:
Probably also sanitizers?

/compiler-rt/*/interception/
/compiler-rt/*/fuzzer/
/llvm/lib/Analysis/StackSafetyAnalysis.cpp

Reply:
why is the first / inconsistent?
e.g. 
/llvm
but 
runtimes/

Reply:
> Sorry, request another line `/llvm/tools/gold/ @llvm/pr-subscribers-llvm-lto`

I've added this.

Reply:
Thanks.
Also
/compiler-rt/lib/fuzzer/ @llvm/pr-subscribers-sanitizer
/compiler-rt/test/fuzzer/ @llvm/pr-subscribers-sanitizer

@cferris1000  It's either sanitizers or a new list
/compiler-rt/lib/scudo/ @llvm/pr-subscribers-sanitizer
/compiler-rt/test/scudo/ @llvm/pr-subscribers-sanitizer